### ISSUE TITLE
Scholarships and 4th year

### ISSUE THREAD

**Issue by garthtarr** [IBS-AR/IBS-AR.github.io#8]
This isn't really a "website" issue, but I just got an email from a student in their 4th year of a combined degree who is keen to apply. I think the issue here is that the scholarship page says "Honours, 4th year, or coursework Masters"... I think the intention is to restrict to "higher" level studies in statistics, not just being in an extended degree program.

Should we say that students need to have "completed" the equivalent of an undergraduate degree and are now pursuing further studies in statistics (e.g. honours or coursework masters)

**Comment by rogerssam** [comment]
Yeah, I think it's intended to encourage further studies, so I don't think a 4th year engineering/stats (for example) student should be eligible.

**Comment by emitanaka** [comment]
I agree as well
